src/test/java/uk/gov/justice/probation/courtcaseservice/controller/OffenderController_CustodyIntTest.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.probation.courtcaseservice.controller;
+
+
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import uk.gov.justice.probation.courtcaseservice.BaseIntTest;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static uk.gov.justice.probation.courtcaseservice.testUtil.TokenHelper.getToken;
+
+class OffenderController_CustodyIntTest extends BaseIntTest {
```

Comment:
Could possibly add an integration test for situation when the NOMS number is not found

Reply:
That's what [this one](https://github.com/ministryofjustice/court-case-service/blob/d31a86f04c14d0bd458aa0a1ebba1ade3fc70ff1/src/test/java/uk/gov/justice/probation/courtcaseservice/controller/OffenderController_CustodyIntTest.java#L59) does I think. Is that what you mean?